src/components/action-bar/action-bar.scss

```diff
@@ -37,6 +37,10 @@ limel-action-bar-button {
     min-width: 0;
 }
 
+.invisible {
```

Comment:
```suggestion
.is-invisible {
```

Reply:
we typically call such classes this way

Reply:
I think this class is unnecessary. We should instead add a `visibility: hidden;` style on the item itself. That'll also ensure that the item cannot receive focus, such as when navigating through tab indexes. I'll add a fixup